### PR TITLE
update project guidelines

### PR DESCRIPTION
**Pull request by percyliang**
Just took a pass and made some local improvements.

### CODE REVIEW

File: project.html [hunk 1]
```diff
@@ -282,8 +293,12 @@
 
 <p>
 <span class="header">Grading rubric</span>
+
+Your project will be graded on the following dimensions:
 <ul>
-  <li><b>Task definition</b>: is the task precisely defined and does the formulation make sense?
+  <li><b>Task definition</b>: is the task precisely defined and is the motivation for the task clear?
+    In other words, does the world somehow become a better place if your project were successful?
```
Comment (rpryzant) [inline project.html:300]: Can we change to "...become a better place if your project *(or the vision it hopes to realize)* were successful?

I think it's okay if projects have good intentions, even if they don't directly solve a problem.

File: project.html [hunk 2]
```diff
@@ -293,11 +308,10 @@
   (i.e., would this be publishable at a good conference)?
 </ul>
 <p>
-Regardless of the group size, all groups must do the same basic amount of work
```
Comment (rpryzant) [inline project.html:296]: Did you mean to delete this part? It seems still relevant.

Comment (percyliang) [reply project.html:296]: Contradicts another sentence.

File: project.html (hunk 1 above)
Comment (percyliang) [reply project.html:300]: I think this statement doesn't require success...it requires (success => better place).

**Pull request merged**